core/src/main/java/org/stellar/anchor/paymentservice/Payment.java

```diff
@@ -7,6 +7,7 @@
 @Data
 public class Payment {
   String id;
+  String txHash;
```

Comment:
Can we generalize this to `idTag` like the `Account` class? Or maybe just `secondaryId`?

For payments that don't involve Stellar it doesn't make sense to have this specific `txHash` attribute in my mind.